new/LIT7103PrayOTprophets.xml

```diff
@@ -0,0 +1,112 @@
+                    <ref type="work" corresp="LIT2265salota"/> or <ref type="work" corresp="LIT2250salota"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="OldTestament"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-10-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <bibl><ptr target="bm:Strelcyn1949PriereFalacha"/><citedRange unit="page">64-75</citedRange></bibl>
+            </div>
+            <div type="edition">
+                <note>Incipits and order of the prayers are taken from <ref type="mss" corresp="BLorient12859"/></note> 
+                <div type="textpart" xml:id="Moses">
+                    <label>Prayer attributed to Moses</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ሙሴ፡ ገበርከ፡ እንዘ፡ ይብል፡ ጋዲን፡ ፈሊዖ፡ <gap reason="ellipsis"/></q>
+                </div>
+                <div type="textpart" xml:id="Joshua">
+                    <label>Prayer attributed to Joshua</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ኢያሱ፡ ወልደ፡ ነዌ፡ ውስተ፡ ሀገረ፡ ገባዖን፡ እንዘ፡ ይብል፡ ደርታን፡</q>
+                </div>
+                <div type="textpart" xml:id="Elijah">
+                    <label>Prayer attributed to Elijah</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ኤልያስ፡ ስመከ፡ እንዘ፡ ይብል፡ አቅርያኖስ፡</q>
+                </div>
+                <div type="textpart" xml:id="Daniel">
+                    <label>Prayer attributed to Daniel</label>
+                    <q xml:lang="gez">ወሶበ፡ <supplied reason="omitted" resp="PRS8999Strelcyn">ጸርሐ፡ </supplied>ዳንኤል፡ ስመከ፡ እንዘ፡ ይብል፡ 
+                        ዝራኤል፡</q>
+                </div>
+                <div type="textpart" xml:id="Manasseh">
+                    <label>Prayer attributed to Manasseh</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ምናሴ፡ ስመከ፡ እንዘ፡ ይብል፡ ቀለኤል፡</q>
+                </div>
+                <div type="textpart" xml:id="ThreeChildren">
+                    <label>Prayer attributed to the Three Holy Children in the Furnace (ʾĀnānyā, ʾĀzāryā and Misāʾel)</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሑ፡ አናንያ፡ ወአዛርያ፡ ወሚሳኤል፡ እንዘ፡ ይብሉ፡ ዳፋኤል፡</q>
+                </div>
+                <div type="textpart" xml:id="David">
+                    <label>Prayer attributed to David</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ስመከ፡ ዳዊት፡ እንዘ፡ ይብል፡ በለማዊፒሲል፡</q>
+                </div>
+                <div type="textpart" xml:id="Ezekiel">
```

Comment:
Canticles have LIT2258salota (NB Hezekiah/Ezekias the King is **not** Ezekiel the Prophet, the Odes and your text have Hezekiah, ሕዝቅያስ, not ሕዝቅኤል)